src/jit/jitconfigvalues.h

```diff
@@ -0,0 +1,193 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
The fundamental issue here is that Jit-EE call getIntConfigValue()/getStringConfigValue() has hardcoded config options disregarding the ones specified in clrconfigvalues.h.  We need to parameterize these routines and also allow config options to be specified against a config setting.  This would essentially mean copying config options from clrconfigvalues.h to jitconfigvalues.h.  Rather i would suggest to strive for a single copy for jit config values.